src/server/scripts/Spells/spell_item.cpp

```diff
@@ -2831,6 +2831,76 @@ class spell_item_toy_train_set_pulse : public SpellScriptLoader
     }
 };
 
+enum DeathChoiceSpells
+{
+    SPELL_DEATH_CHOICE_NORMAL_AURA      = 67702,
+    SPELL_DEATH_CHOICE_HEROIC_AURA      = 67771,
+    SPELL_DEATH_CHOICE_NORMAL_STRENGHT  = 67708,
+    SPELL_DEATH_CHOICE_NORMAL_AGILITY   = 67703,
+    SPELL_DEATH_CHOICE_HEROIC_STRENGHT  = 67773,
+    SPELL_DEATH_CHOICE_HEROIC_AGILITY   = 67772
```

Comment:
@Exodius There is a third option: order enums descending by value (I usually do that). ;)